rcswitch/user_doc.rst

```diff
@@ -0,0 +1,134 @@
+.. index:: Plugins; rcswitch (Brennenstuhl RCS 1000 N)
+.. index:: rcswitch
+
+================
+Plugins/rcswitch
+================
+
+
+Anforderungen
+-------------
+Das rcswitch Plugin beötigt folgenden Anforderungen an Hanrdware- und Software-Komponenten.
+
+Notwendige Hardware
+~~~~~~~~~~~~~~~~~~~
+
+* Raspberry Pi
+* 433 MHz Sende-Module
+* schaltbare Steckdose über 433 MHz (z.B.: Brennenstuhl RCS 1000 N)
+
+Notwendige Software
+~~~~~~~~~~~~~~~~~~~
+
+* die python Library `rpi-rf`
+
+
+Unterstützte Geräte
+~~~~~~~~~~~~~~~~~~~
+
+* Brennenstuhl RCS 1000 N
+
+
+Konfiguration
```

Comment:
Unter Konfiguration sollte ein Satz mit dem Link zur automatisch generierten Seite mit der Konfigurationsdokumentation stehen. 

So etwas wir:

Diese Plugin Parameter und die Informationen zur Item-spezifischen Konfiguration des Plugins sind
unter :doc:`/plugins_doc/config/rcswitch2` beschrieben.

Reply:
erl. in PR für `rcs1000n`